LocalWiki.php

```diff
@@ -265,6 +265,11 @@ function wfModifyMetaTags( OutputPage $out ) {
 		break;
 	case 'empty':
```

Comment:
Empty should always be the bottom for stylistic reasons (as it's essentially blank, a terminating one)